1.0/docs/devguide/properties.md

```diff
@@ -370,40 +370,59 @@ Example:
 In addition to properties, observers can also observe [paths to sub-properties](#observing-path-changes),
 [paths with wildcards](#deep-observation), or [array changes](#array-observation).
 
-### Observing path changes {#observing-path-changes}
+### Observing sub-property changes {#observing-path-changes}
 
-You can also observe changes to object sub-properties using the 
-`observers` array, by specifying a full path (`user.manager.name`)
-as a function argument.
+To observe changes in object sub-properties:
 
-Example:
-
-    Polymer({
+1.  Initialize your object to a default value.
+2.  Define an `observers` array.
+3.  For each object sub-property that you want to observe, add an element to
```

Comment:
YES! "element" is this context was irking me. Thank you. 

:heavy_check_mark: changed element (in context of arrays) to item
:heavy_check_mark: changed instructions to talk about a single item